api/src/main/java/uk/gov/dhsc/htbhf/claimant/factory/CardRequestFactory.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.claimant.factory;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.converter.AddressToAddressDTOConverter;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.model.AddressDTO;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
+
+/**
+ * Creates a {@link CardRequest} instance from a given {@link Claim}.
+ */
+@Service
+@AllArgsConstructor
+public class CardRequestFactory {
+
+    private AddressToAddressDTOConverter addressConverter;
+
+    public CardRequest createCardRequest(Claim claim) {
+        AddressDTO address = addressConverter.convert(claim.getClaimant().getCardDeliveryAddress());
```

Comment:
Minor, but you should assert that the Claim isn't null here too, plus add a test for it.